db/migrate/20220603145240_change_user_gender_enum.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeUserGenderEnum < ActiveRecord::Migration[7.0]
+  def up
+    execute "ALTER TABLE users ALTER gender DROP DEFAULT;"
+    change_column :users, :gender, :integer, using: 'gender::integer'
+  end
```

Comment:
Same here, let's define a `change` method. Also if you only include a `up` method, that will make your migration irreversible.

BTW, can we use [this](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_default) instead of running a plain SQL query for removing the default attribute?

Reply:
Because I had data in this field, and when I try to change by enumeration, I have problems with the database.

Reply:
Since this is not in production yet (deployed), you can delete your records locally and just change the data type.